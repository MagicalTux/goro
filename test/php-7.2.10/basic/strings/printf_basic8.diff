--
     int(2)
     
     -- Calling printf() with two arguments --
012- 21 37777777431
013- int(14)
012+ 21 -347
013+ int(7)
     
     -- Calling printf() with three arguments --
016- 21 37777777431 567
017- int(18)
016+ 21 -347 567
017+ int(11)
     ===DONE===
