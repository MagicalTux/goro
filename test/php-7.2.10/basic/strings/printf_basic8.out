*** Testing printf() : basic functionality - using octal format ***

-- Calling printf() with no arguments --
format
int(6)

-- Calling printf() with one arguments --
21
int(2)

-- Calling printf() with two arguments --
21 -347
int(7)

-- Calling printf() with three arguments --
21 -347 567
int(11)
===DONE===